src/main/kotlin/uk/gov/dwp/dataworks/services/ConfigurationResolver.kt

```diff
@@ -62,6 +62,7 @@ enum class ConfigKey(val key: String, val isList: Boolean) {
     ECS_CLUSTER_NAME("orchestrationService.ecs_cluster_name", false),
     USER_CONTAINER_URL("orchestrationService.user_container_url", false),
     USER_CONTAINER_PORT("orchestrationService.user_container_port", false),
+    JUPYTER_S3_ARN("orchestrationService.jupyterhub_bucket_arn", false),
```

Comment:
We might need to remove the `jupyterhub_bucket_arn` from TF now that it's not needed though

Reply:
I am confused by this - it is needed, and is being used.

Reply:
That's me being an idiot - I meant `orchestrationService.jupyterhub_bucket`